acf-autosize.php

```diff
@@ -2,8 +2,8 @@
 /*
 Plugin Name: ACF Autosize
 Plugin URI: https://wordpress.org/plugins/acf-autosize/
-Description: A wordpress plugin to automatically resize and improve upon wysiwyg and textarea fields in Advanced Custom Fields. <strong>Please note: Version 2 will automatically apply autosize to all fields. It will be possible to deactivate autosizing via class though.</strong>
-Version: 1.3.3
+Description: A wordpress plugin to automatically resize and improve upon wysiwyg and textarea fields in Advanced Custom Fields.
+Version: 2.0.0
```

Comment:
Who would have thought there would be a 2.0.0 :-)

Reply:
whoop!